agent/pkg/podman/podman.go

```diff
@@ -184,7 +186,9 @@ func (s *PodmanClient) RunContainer(
 	args = append(args, "--env", "NVIDIA_VISIBLE_DEVICES")
 	args = append(args, "--env", "ROCR_VISIBLE_DEVICES")
 	args = append(args, "--env", "HIP_VISIBLE_DEVICES")
-
+	if s.debug {
+		args = append(args, "--env", "DET_DEBUG=1")
```

Comment:
i guess i was thinking to set this from the experiment or command config but this works too, given how we are running this.